ttm/releaser.py

```diff
@@ -296,7 +305,7 @@ def _release(self, set_release=None):
 
         for image in self.project.image_products:
             self.release_package(self.project.name, image.package, set_release=set_release,
-                                 repository=self.project.product_repo,
+                                 repository=getattr(self.project, 'totest_images_repo', self.project.product_repo),
```

Comment:
hmm...it's needed another solution, otherwises it might break Factory release from `images` to `appliances`..